packages/terra-embedded-content-consumer/src/terra-dev-site/test/embedded-content-consumer/providers/CustomEventsProvider.test.jsx

```diff
@@ -1,38 +0,0 @@
-import React from 'react';
```

Comment:
Renamed to src/terra-dev-site/provider/embedded-content-consumer/providers/CustomEventsProvider.provider.jsx